### PR TITLE
Added implementation for the Pagination feature, added swager-ui dependencies.

### PR DESCRIPTION
**Pull request by MinoDentori**
1. Added dependencies for the OpenDoc swagger specification.
2. Specified path to API docs in application properties.
3. Change getAll endpoint to accept Paginable objects as a parameter.
4. Added getAll(Paginable paginable) method to the BookService and implementation to the BookServiceImpl.
5. Fixed code style mistakes.
6. Tested via Postman.

### CODE REVIEW

File: src/main/java/mate/academy/onlinebookstore/controller/BookController.java [hunk 1]
```diff
@@ -18,39 +21,46 @@
...
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.getAll();
+    @Operation(summary = "Get all books")
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.getAll(pageable);
     }
 
     @GetMapping("/{id}")
+    @Operation(summary = "Find book by id")
```
Comment (HovorukhaBohdan) [inline src/main/java/mate/academy/onlinebookstore/controller/BookController.java]: I think we should provide description for `@Operation`. Like this:
image

**Review by HovorukhaBohdan — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/onlinebookstore/controller/BookController.java (hunk 1 above)
Comment (MinoDentori) [reply src/main/java/mate/academy/onlinebookstore/controller/BookController.java]: fixed

File: src/main/java/mate/academy/onlinebookstore/service/BookService.java [hunk 2]
```diff
@@ -4,17 +4,21 @@
...
 import mate.academy.onlinebookstore.dto.BookSearchParametersDto;
 import mate.academy.onlinebookstore.dto.CreateBookRequestDto;
+import org.springframework.data.domain.Pageable;
 
 public interface BookService {
     BookDto createBook(CreateBookRequestDto bookDto);
 
     List<BookDto> getAll();
 
+    List<BookDto> getAll(Pageable pageable);
```
Comment (kshuryhin) [inline src/main/java/mate/academy/onlinebookstore/service/BookService.java]: just wondering if you need both these methods for fetching, although you're using in the API only pageable one

**Review by kshuryhin — CHANGES_REQUESTED** [review]
great job, just one point to consider

File: src/main/java/mate/academy/onlinebookstore/service/BookService.java (hunk 2 above)
Comment (MinoDentori) [reply src/main/java/mate/academy/onlinebookstore/service/BookService.java]: really, no usages for getAl(), cleared.

**Review by kshuryhin — APPROVED** [review]
looks good

**Pull request merged**